language/move-vm/runtime/src/loader.rs

```diff
@@ -753,6 +753,17 @@ impl Loader {
         self.verify_ty_args(func.type_parameters(), &type_arguments)
             .map_err(|e| e.finish(Location::Module(module_id.clone())))?;
 
+        if !func.type_parameters.is_empty() {
+            if let Some(max_count) = self.vm_config.verifier.max_function_instantiation_size {
+                let mut type_params = vec![];
+                for idx in 0..func.type_parameters.len() {
+                    type_params.push(Type::TyParam(idx))
+                }
```

Comment:
For fancy pants points, consider `let type_params = (0..func.type_parameters.len()).map(Type::TyParam).collect::<Vec<_>>();`